OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -27,30 +27,6 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <Optimize>false</Optimize>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)' == 'Red Alert' OR '$(RunConfiguration)' == 'Red Alert' ">
```

Comment:
It seems that MonoDevelop/vs:mac don't support `launchSettings.json` after all. Can we please keep the `$(RunConfiguration)`s for that?